bgpd/bgp_io.c

```diff
@@ -23,7 +23,7 @@
 #include <zebra.h>
 #include <pthread.h>		// for pthread_mutex_unlock, pthread_mutex_lock
 
-#include "frr_pthread.h"	// for frr_pthread_get, frr_pthread
```

Comment:
comment should just read `frr_pthread` now since struct is still used

Reply:
TBH I'm very much not a fan of these comments either way... they try to provide information that:

- can be automatically extracted with some effort
- is prone to be outdated by oversight
- doesn't really have a lot of impact to a developer working on the code
- isn't consistently there throughout the codebase

So that's why I stripped the comment on the line I touched.  If I were to change anything here I would rather strip the comments from the other headers too.

Reply:
¯\_(ツ)_/¯